packages/forms-web-app/src/dynamic-forms/dynamic-components/radio/index.njk

```diff
@@ -41,6 +41,10 @@
           attributes: { "data-cy":"button-save-and-continue"}
         }) }}
       </form>
+
+      <p class="govuk-body">
+        <a href="{{listLink}}" class="govuk-link">Return to questionnaire homepage</a>
```

Comment:
maybe for future, but do we want the displayed text to be fixed, or something that can be passed into the template?

Reply:
I'll move the whole thing to the template